src/functionalTest/java/uk/gov/hmcts/reform/sscs/functional/OcrFormValidationFunctionalTest.java

```diff
@@ -31,7 +31,7 @@ public void should_validate_ocr_data_and_return_success(String formType) throws
         Response response = validateOcrEndpointRequest(json, formType, OK.value());
 
         OcrValidationResponse validationResponse = response.getBody()
-            .as(OcrValidationResponse.class, ObjectMapperType.GSON);
+            .as(OcrValidationResponse.class, ObjectMapperType.JACKSON_2);
```

Comment:
Needed the following change

https://github.com/rest-assured/rest-assured/wiki/Usage#content-type-based-serialization